### PR TITLE
Day3: Header headline section

### PR DESCRIPTION
**Pull request by io-oseinana**
# My Portfolio Webpage

In this project I:

-  Build a Header for the homepage
-  Build a Headline section for page
-  Added linters to check for Errors in code

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  margin-top: 10px;
+  color: #172b4d;
+}
+
+.job-title {
+  color: #36b37e;
+}
+
+.text {
+  font-weight: 400;
```
Comment (ridwanediallo) [inline css/main.css:58]: - [ ] Please make that your text section has the right margins to the left and the right.
## Your section 👇️
Screenshot from 2022-09-21 19-43-26

## the figma design 👇️

Screenshot from 2022-09-21 19-42-17

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  }
+
+  .header {
+    margin: 20px;
+  }
+
+  .headline-sec {
+    display: flex;
+    flex-direction: column;
+    background-image: url(/img/bg-header-illsutration-img/header-illsutration-mobile.svg);
```
Comment (ridwanediallo) [inline css/main.css:97]: - [ ] Kindly make sure the background image is properly positioned to the right. ([See the Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468))

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @io-oseinana , 

AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔
## Optional suggestions


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @io-oseinana 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**